packages/plugins/graphql/server/services/internals/scalars/date.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const { GraphQLDate } = require('graphql-iso-date/dist');
+
+const parseAndCast = parseFn => (...args) => {
+  const parsedValue = parseFn(...args);
+
+  if (parsedValue instanceof Date) {
+    return parsedValue.toISOString().split('T')[0];
```

Comment:
After all I didn't use `formatISO` from `date-fns` as `formatISO` was taking the local timezone into account whereas `parsedValue` was not created using the local timezone (`parsedValue.toISOString()` always has hours and minutes set to 0). When using a timezone in america, the date was one day prior to the real one.